libraries/botbuilder-applicationinsights/src/telemetryInitializerMiddleware.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+
+import { BotTelemetryClient, NullTelemetryClient, TelemetryLoggerMiddleware } from 'botbuilder-core';
+import { Middleware } from 'botbuilder-core';
+import { TurnContext } from 'botbuilder-core';
+import { TelemetryClient } from 'applicationinsights';
+import { CorrelationContext } from 'applicationinsights/out/AutoCollection/CorrelationContextManager';
+import * as appInsights from 'applicationinsights';
+
+/**
+ * Middleware for storing the incoming activity to be made available to Application Insights and optionally run the TelemetryLoggerMiddleware.
+ * Uses the botTelemetryClient interface.
+ */
+export class TelemetryInitializerMiddleware implements Middleware {
+
+    private readonly _telemetryClient: BotTelemetryClient;
+
+    // tslint:disable:variable-name
+    private readonly _logActivityTelemetry: boolean;
+    private readonly _logPersonalInformation: boolean;
+    // tslint:enable:variable-name
+
+    /**
+     * Initializes a new instance of the TelemetryInitializerMiddleware class.
+     * @param _telemetryClient The TelemetryClient used by the TelemetryLoggerMiddleware for logging activity telemetry.
+     * * @param _logActivityTelemetry (Optional) Enable/Disable logging of activity telemetry.
+     * * @param _logPersonalInformation (Optional) Enable/Disable logging original message name within Application Insights.
```

Comment:
these should not have _ in front of them

Reply:
Removed